app/src/main/java/it/niedermann/nextcloud/deck/persistence/sync/SyncManager.java

```diff
@@ -745,7 +745,7 @@ public void onError(Throwable throwable) {
             Account account = dataBaseAdapter.getAccountByIdDirectly(accountId);
             FullBoard board = dataBaseAdapter.getFullBoardByLocalIdDirectly(accountId, stack.getBoardId());
             FullStack fullStack = new FullStack();
-            // TODO set stack order to (highest stack-order from board) + 1 and remove logic from caller
```

Comment:
Did you remove the logic from the caller?

Reply:
Done in https://github.com/stefan-niedermann/nextcloud-deck/pull/529/commits/c93c9901eb650a2cf38b6749fe7cfb3426a14bdb